src/objects/Volunteer_Hours__c.object

```diff
@@ -409,9 +408,143 @@
         </displayedFields>
         <label>VolunteersJobListingFS</label>
     </fieldSets>
+    <fieldSets>
+        <fullName>VolunteersReportHoursFS</fullName>
+        <availableFields>
+            <field>Contact__c</field>
```

Comment:
Contact, CreatedById, CreatedDate, LastActivityDate, Id, Full name, IsDeleted, LastModifiedById, LastModifiedDate, SystemModstamp should not be included in the field set, since these fields are read only and typically set either the page, or Salesforce.